spec/plugins/hmac-auth/access_spec.lua

```diff
@@ -0,0 +1,150 @@
+local spec_helper = require "spec.spec_helpers"
+local http_client = require "kong.tools.http_client"
+local cjson = require "cjson"
+local sha1 = require "sha1"
+local base64 = require "base64"
+
+local json_decode = cjson.decode
+local os_date = os.date
+local base64_encode = base64.encode 
```

Comment:
Just to understand how its fine to use global optimisations at one but non readable at another place. isn't it better to be optimise it everywhere given alias used for function is very descriptive?